RealmSwift/Combine.swift

```diff
@@ -503,14 +544,57 @@ public enum Publishers {
         /// This publisher emits Void.
         public typealias Output = Void
 
+        private let collection: Collection
+
+        internal init(_ collection: Collection) {
+            self.collection = collection
+        }
+
+        /// Allows you to capture the `NotificationToken` produced by observing a Realm Collection.
+        /// - Parameters:
+        ///   - object: The object which owns the `NotificationToken`.
+        ///   - keyPath: The KeyPath of the `NotificationToken`.
```

Comment:
The object which the notification token is written to, and the keypath which is it written to.